### PR TITLE
"Free Trial" and "Support" nav buttons

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Redesigned "Free trial" and "Contact support" buttons. 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [x] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-394

### CODE REVIEW

File: docs/contributing/index.md [hunk 1]
```diff
@@ -1,18 +1,25 @@
 ---
 slug: /contributing
-title: Contribute to Sumo Docs
+title: Contributing to Sumo Logic Docs
```
Comment (kimsauce) [inline docs/contributing/index.md:3]:
```suggestion
title: Contribute to Sumo Docs
```

File: docs/contributing/index.md [hunk 2]
```diff
@@ -8,11 +8,18 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+We're a community of developers and practitioners dedicated to building reliable and secure modern apps. At Sumo Logic, we believe in the power of collaboration and community-driven innovation. Our open-source projects go beyond code—they're about fostering a vibrant, inclusive community where every contribution, big or small, makes an impact.
+
+Whether you’re fixing bugs, adding features, writing documentation, or sharing feedback, your contributions help others learn and grow within the Sumo Logic ecosystem. From typo fixes to new docs, we welcome all forms of participation.
+
+Our open-source [documentation](https://github.com/SumoLogic/sumologic-documentation) project is hosted on GitHub, where you’ll find everything you need to get started — guidelines on creating files, writing in Markdown, using templates, and submitting pull requests.
+
+Whether you’re a seasoned developer or just starting, join us in building tools that help everyone better understand and optimize their systems. Ready to contribute? Explore our [contributing guide](https://help.sumologic.com/docs/contributing/) and get involved today!
```
Comment (JV0812) [inline docs/contributing/index.md:19]:
```suggestion
Whether you’re a seasoned developer or just starting, join us in building tools that help everyone to better understand and optimize their systems. Ready to contribute? Explore our [contributing guide](https://help.sumologic.com/docs/contributing/) and get involved today!
```

File: docs/contributing/index.md [hunk 3]
```diff
@@ -8,11 +8,18 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+We're a community of developers and practitioners dedicated to building reliable and secure modern apps. At Sumo Logic, we believe in the power of collaboration and community-driven innovation. Our open-source projects go beyond code—they're about fostering a vibrant, inclusive community where every contribution, big or small, makes an impact.
+
+Whether you’re fixing bugs, adding features, writing documentation, or sharing feedback, your contributions help others learn and grow within the Sumo Logic ecosystem. From typo fixes to new docs, we welcome all forms of participation.
+
+Our open-source [documentation](https://github.com/SumoLogic/sumologic-documentation) project is hosted on GitHub, where you’ll find everything you need to get started — guidelines on creating files, writing in Markdown, using templates, and submitting pull requests.
```
Comment (JV0812) [inline docs/contributing/index.md:17]:
```suggestion
Our open-source [documentation](https://github.com/SumoLogic/sumologic-documentation) project is hosted on GitHub, you can find all the resources to get started — guidelines on creating files, writing in Markdown, using templates, and submitting pull requests.
```

File: docs/contributing/index.md [hunk 4]
```diff
@@ -8,11 +8,18 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+Welcome to Sumo Logic Open Source!
 
-Sumo Logic Documentation is an Open Source project available as a [GitHub repository](https://github.com/SumoLogic/sumologic-documentation). This section will grow as we answer questions and provide support for submissions and new content. Here you'll find information on creating files, writing in markdown, using our templates, submitting pull requests (PRs), and more.
+We're a community of developers and practitioners dedicated to building reliable and secure modern apps. At Sumo Logic, we believe in the power of collaboration and community-driven innovation. Our open-source projects go beyond code—they're about fostering a vibrant, inclusive community where every contribution, big or small, makes an impact.
+
+Whether you’re fixing bugs, adding features, writing documentation, or sharing feedback, your contributions help others learn and grow within the Sumo Logic ecosystem. From typo fixes to new docs, we welcome all forms of participation.
```
Comment (JV0812) [inline docs/contributing/index.md:15]:
```suggestion
Whether you’re fixing bugs, adding features, writing documentation, or sharing feedback, your contributions help others to learn and grow within the Sumo Logic ecosystem. From typo fixes to new docs, we welcome all forms of participation.
```

**Review by JV0812 — APPROVED** [review]
Minor suggestion.

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@kimsauce, when I click "Contribute to Docs", the Support button changes to this:
Screenshot 2024-09-19 at 8 49 59 AM

Also, the dark down arrow in the light mode is not readable. Could you make the down arrow blue so that it will show up in both light and dark mode?

File: docs/contributing/index.md (hunk 2 above)
Comment (kimsauce) [reply docs/contributing/index.md:19]: Thank you! I think I'm going to stet the original version for a more conversational tone.

**Comment by kimsauce** [comment]
> @kimsauce, when I click "Contribute to Docs", the Support button changes to this...
> 
> Also, the dark down arrow in the light mode is not readable. Could you make the down arrow blue so that it will show up in both light and dark mode?

Fixed the "Contribute to Docs" issue and I made the text + arrow color both white in light + dark mode. Thanks for your review!
Screenshot 2024-10-10 at 4 48 34 PM


Screenshot 2024-10-10 at 4 48 23 PM

**Pull request merged**